src/sidebar.ts

```diff
@@ -197,7 +196,60 @@ const quickstart: SidebarItemsConfig = [
     label: "Migrate to Ory",
     collapsed: false,
     collapsible: false,
-    items: ["migrate-to-ory/index", "migrate-to-ory/auth0"],
+    items: [
```

Comment:
This sidebar is quite convoluted. I suggest either making this its own sidebar or somehow flattening the items a little so that they're not 4 levels deep.